service/iot-service-samples/digitaltwin-service-samples/temperature-controller-service-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/TemperatureController.java

```diff
@@ -142,10 +145,23 @@ private static void InvokeMethodOnComponent() throws IOException {
         String commandInput = ZonedDateTime.now(ZoneOffset.UTC).minusMinutes(5).format(DateTimeFormatter.ISO_DATE_TIME);
 
         DigitalTwinInvokeCommandRequestOptions options = new DigitalTwinInvokeCommandRequestOptions();
-        ServiceResponseWithHeaders<DigitalTwinCommandResponse, DigitalTwinInvokeCommandHeaders> commandResponse = client.invokeComponentCommandWithResponse(digitalTwinid, componentName, commandName, commandInput, options);
-        System.out.println("Command " + commandName + ", payload: " + prettyString(commandResponse.body().getPayload()));
-        System.out.println("Command " + commandName + ", status: " + commandResponse.body().getStatus());
-        System.out.println("Command " + commandName + ", requestId: " + commandResponse.headers().getRequestId());
+        try {
+            ServiceResponseWithHeaders<DigitalTwinCommandResponse, DigitalTwinInvokeCommandHeaders> commandResponse = client.invokeComponentCommandWithResponse(digitalTwinid, componentName, commandName, commandInput, options);
+            System.out.println("Command " + commandName + ", payload: " + prettyString(commandResponse.body().getPayload()));
+            System.out.println("Command " + commandName + ", status: " + commandResponse.body().getStatus());
+            System.out.println("Command " + commandName + ", requestId: " + commandResponse.headers().getRequestId());
+        } catch (RestException ex)
```

Comment:
👍 
For update, don't we need something similar?

Reply:
Good call, you are right for update too. I will update sample in next PR for update.